pkg/collector/systemlogs_collector.go

```diff
@@ -15,11 +15,11 @@ type SystemLogsCollector struct {
 var _ interfaces.Collector = &SystemLogsCollector{}
 
 // NewSystemLogsCollector is a constructor
-func NewSystemLogsCollector(exporter interfaces.Exporter) *SystemLogsCollector {
+func NewSystemLogsCollector(exporters []interfaces.Exporter) *SystemLogsCollector {
 	return &SystemLogsCollector{
 		BaseCollector: BaseCollector{
 			collectorType: SystemLogs,
-			exporter:      exporter,
+			exporters:     exporters,
 		},
 	}
 }
```

Comment:
interface signature change to an array, no reason it had to be a single exporter